common/src/main/java/com/wynntils/core/features/AbstractConfigurable.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features;
+
+import com.google.common.base.CaseFormat;
+import com.wynntils.core.config.ConfigHolder;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public abstract class AbstractConfigurable implements Configurable {
```

Comment:
Why is `getConfigOptions` (unfiltered version) missing? The interface should have it too.

Reply:
It is not missing?

Reply:
I mean we should add it. I know the interface does not have the method too.

Reply:
```java
    /** Returns all configurable options  that should be visible to the user */
    List<ConfigHolder> getConfigOptions();
```
in `Configurable`?

Reply:
I am blind.